src/internal/channel.js

```diff
@@ -13,9 +13,21 @@ export function emitter() {
     return () => remove(subscribers, sub)
   }
 
+  function emitAll(items) {
+    for (var i = 0, len = items.length; i < len; i++) {
+      emit(items[i])
+    }
+  }
+
   function emit(item) {
+    if (item === undefined) {
```

Comment:
wondering also if `processAction` could be made a concern of `emit` itself, right now you are passing already processed actions into `emit`